src/Analyzers/Core/CodeFixes/CodeFixesResources.resx

```diff
@@ -120,8 +120,8 @@
   <data name="Add_file_header" xml:space="preserve">
     <value>Add file header</value>
   </data>
-  <data name="Fix_Name_Violation_colon_0" xml:space="preserve">
-    <value>Fix Name Violation: {0}</value>
+  <data name="Fix_Name_Violation" xml:space="preserve">
+    <value>Fix Name Violation</value>
```

Comment:
If the resource is being changed anyway, I think the casing should be fixed to be consistent with other codefix titles.

```suggestion
  <data name="Fix_name_violation" xml:space="preserve">
    <value>Fix name violation</value>
```